src/visualization/plot_recipes.jl

```diff
@@ -478,21 +478,29 @@ function to `solution_variables`.
 twice the number of solution DG nodes are used for visualization, and if set to `0`,
 exactly the number of nodes in the DG elements are used.
 
-
+When visualizing data from a two-dimensional simulation, a 1D stripe is extracted for plotting.
+`stripe_axis` specifies the axis stripped from the plane and may be `:x`, or `:y`. The
+point on the stripe axis where it intersects with the orthogonal axis is given in `stripe_axis_intercept`.
+Both of these values are ignored when visualizing 1D data.
```

Comment:
To me, it feels like `line_axis` and `line_axis_intercept` are not very intuitive concepts, but I have a hard time coming up with better terms (that, preferably, would also work for creating 2D slices from 3D data, such that is consistent - `slice_axis` and `slice_axis_intercept` aren't fully self-explanatory either). We should really think a little bit about this...

Reply:
Maybe
3D -> 2D: `slice` with possible values `:xy`, `:xz`, and `:yz`, together with `slice_intercept`
2D -> 1D: `slice` with possible values `:x`, `:y`, together with `slice_intercept`

What do you think? I'd also like to get @erik-f's input on this if possible, since it was him (IIRC) who coined the original terms.

Reply:
Is this supposed to work only for axis parallel lines? Otherwise, maybe start and end points of the line (like in ParaView) could be more intuitive.

Reply:
If not, I think your suggestion to keep it consistent with 3D sounds good to me, @sloede

Reply:
@erik-f This is a good point! I don't know yet how we'll handle slices that are not orthogonal to the axes, but I would assume that the easiest way to specify them is as in Paraview, where you typically give a point and a normal direction.

Reply:
@FelipeSantillan After getting some input from @ranocha, let's proceed as discussed in #563:
* use `slice` for the slice plane/line, which for 3D -> 2D can be `:xy`, `:xy`, or `:yz`, and for [23]D -> 1D can be `:x`, `:y`, or `:z`
* use `point` to specify a 2D/3D point on which the `slice` is located
Essentially this follows the suggestion in https://github.com/trixi-framework/Trixi.jl/issues/563#issuecomment-826039196.

It means that some information from `point` is discarded, but it would be consistent across dimensions and extensible to non-orthogonal slice placement.